index.html

```diff
@@ -3954,6 +3954,17 @@ <h4>equivalentId</h4>
 <code><a>equivalentId</a></code> DID and the <code>id</code> property DID.
                   </p>
                 </div>
+                
+                <div class="issue" title="Observance of equivalentId recommendations">
+                  <p>
+If a resolving party does not retain the values from the <code>id</code> and
+<code><a>equivalentId</a></code> properties and ensure any subsequent
+interactions with any of the values they contain are correctly handled as
+logically equivalent, there may be negative or unexpected issues that 
+arise related to user experience. You are strongly advised to observe the 
```

Comment:
```suggestion
arise. Implementers are strongly advised to observe the 
```

The negative issues aren't just related to user experience, they're also related to security. Make the 'You' party more explicit.